lib/src/logger_extension.dart

```diff
@@ -1,9 +1,41 @@
 import 'package:mason_logger/mason_logger.dart';
+import 'package:universal_io/io.dart';
 
 /// Extension on the Logger class for custom styled logging.
 extension LoggerX on Logger {
   /// Log a message in the "created" style of the CLI.
   void created(String message) {
     info(lightCyan.wrap(styleBold.wrap(message)));
   }
+
+  /// Wrap the [text] to fit perfectly within the width of the terminal when
+  /// [print]ed.
+  ///
+  /// To overwrite the width you can use [length].
+  void wrap(
```

Comment:
By any chance do you think we can contribute to MasonLogger with this method?

Reply:
Had a chat with Felix about it but he wants to refrain for now as that might open doors for other features that arent "logging" related. So for now I'll resolve this thread